routing_table/updater.go

```diff
@@ -113,28 +115,37 @@ func (u *updater) Sync() {
 	logger.Debug("fetched-tcp-routes", lager.Data{"num-routes": len(tcpRouteMappings)})
 
 	if err == nil {
-		freshRoutingTable := models.NewRoutingTable(logger)
+		freshRoutingTable := models.NewRoutingTableWithSession(logger, "fresh-routing-table")
```

Comment:
Does a scenario exist where there would be multiple "fresh-routing-table" sessions?  Should this be more dynamically named?